packages/atomic/src/pages/v1.html

```diff
@@ -62,7 +62,7 @@
       }
 
       atomic-search-interface {
-        grid-template-columns: 1fr minmax(150px, 325px) minmax(500px, 925px) 1fr;
```

Comment:
This matches the design more closely and made it easier to test the grid, which I'll open a separate PR for.